src/plugin.js

```diff
@@ -41,6 +41,19 @@
     }
   }
 
+  oldOptions = videojs.Player.prototype.options;
+  videojs.Player.prototype.options = function() {
+    var options = oldOptions.call(this);
+    if (Object.prototype.toString.call(options.children) === '[object Array]') {
+      for (var i = 0; i < options.children.length; i++) {
+        var childName = options.children[i];
+        options.children[childName] = this.getChild(childName);
```

Comment:
Hm... yeah, we probably want to grab the options of the component instead of the component itself.